drivers/tee/optee/rpc.c

```diff
@@ -307,6 +309,48 @@ static void handle_rpc_func_cmd_shm_free(struct tee_context *ctx,
 	arg->ret = TEEC_SUCCESS;
 }
 
+static void handle_rpc_func_cmd_bm_reg(struct optee_msg_arg *arg)
+{
+	u64 size;
+	u64 type;
```

Comment:
@etienne-lms 
size is used here (line 330):
```
memremap(arg->params[0].u.value.b, size, MEMREMAP_WB);
```
so I thought that it's better to have `size` than `arg->params[0].u.value.c;` in func params (for the sake of better readability)

the same for `type`